### PR TITLE
rev_Allow to kill a specific job

### PR DESCRIPTION
**Pull request by apodhrad**
Such as "Refreshing server adapter list"

### CODE REVIEW

**Comment by rawagner** [comment]
Please open PR also for 1.0.x

**Comment by rawagner** [comment]
test this

**Comment by apodhrad** [comment]
this is still under development

**Comment by apodhrad** [comment]
test this

**Comment by rawagner** [comment]
test this

File: plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/condition/JobIsKilled.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	/**
+	 * Constructs JobIsKilled condition using the specified jobs which should be killed.
+	 * 
+	 * @param jobsToBeKilled
+	 *            Jobs which should be killed
+	 */
+	public JobIsKilled(String... jobsToBeKilled) {
+		this.jobsToBeKilled = new HashSet<>();
+		this.killedJobs = new HashSet<>();
+		for (String jobToBeKilled : jobsToBeKilled) {
```
Comment (rawagner) [inline plugins/org.jboss.reddeer.core/src/org/jboss/reddeer/core/condition/JobIsKilled.java]: you can use addAll() instead of for cycle

**Pull request merged**